test/cbmc/proofs/parsing/ProcessReceivedTCPPacket/Makefile.json

```diff
@@ -9,10 +9,12 @@
   "OBJS":
   [
     "$(ENTRY)_harness.goto",
+    "$(FREERTOS_PLUS_TCP)/test/cbmc/stubs/cbmc.goto",
```

Comment:
Why is this needed?

Reply:
To get the source of [safeMalloc](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dev/IPv6_integration/test/cbmc/stubs/cbmc.c#L8).